### PR TITLE
Issue#595 A Task does not show the address

### PR DESCRIPTION
**Pull request by AdManteza**
- [x] Use the view helper to render the address
- [x] Dry up and refactor the `tasks/show` page a bit

### CODE REVIEW

File: app/views/tasks/show.html.erb [hunk 1]
```diff
@@ -21,18 +21,13 @@
   <p><b>Description:</b> <%= @task.description %></p>
 <% end %>
 
-<% if @task.street.present? and @task.city.present? and @task.state.present? and @task.zipcode.present? %>
-  <p>
-    <%= @task.street %><br>
-    <%= @task.city %>, <%= @task.state %> <%= @task.zipcode %>
-  </p>
-<% end %>
+<%= task_address %>
```
Comment (kgf) [inline app/views/tasks/show.html.erb:24]: I like moving this to the helper

File: app/helpers/tasks_helper.rb [hunk 2]
```diff
@@ -14,6 +14,21 @@ def task_priority_label(task)
...
   end
 
+  def has_latitude_longitude?
+    @task.latitude.present? and
+    @task.longitude.present?
+  end
+
+  def task_address
+    city = @task.city
+    city = @task.city + ', ' if @task.city.present?
```
Comment (kgf) [inline app/helpers/tasks_helper.rb]: Should we add the comma if the `@task.state` is missing ?

Comment (AdManteza) [reply app/helpers/tasks_helper.rb]: Good call!

**Comment by AdManteza** [comment]
@kgf Ready for another look!

**Comment by kgf** [comment]
This is regarding #595

**Comment by kgf** [comment]
@AdManteza Thanks for your help !

**Pull request merged**